container.go

```diff
@@ -131,6 +131,10 @@ type ContainerRequest struct {
 	HostConfigModifier      func(*container.HostConfig)                // Modifier for the host config before container creation
 	EnpointSettingsModifier func(map[string]*network.EndpointSettings) // Modifier for the network settings before container creation
 	LifecycleHooks          []ContainerLifecycleHooks                  // define hooks to be executed during container lifecycle
+	// KeepImage describes whether DockerContainer.Terminate should not delete the
+	// container image. Useful for images that are built from a Dockerfile and take a
+	// long time to build. Keeping the image also Docker to reuse it.
+	KeepImage bool
```

Comment:
Hey @lefinal wdyt if we move this new field to the `FromDockerfile` struct? It will be even clearer that this is specific to images that are built from Dockerfiles.

Reply:
Yes, makes absolutely sense.